### DISCUSSION TITLE
Install Not Working

### DISCUSSION THREAD

**Discussion by DanDM84** [UnchartedBull/OctoDash#1333] [Q&A] [answered]
I'm new to bash/wget, etc. I have tried to run the following over SSH, and also directly logged in as "pi" on the raspberry pi:  

bash <(wget -qO- https://github.com/UnchartedBull/OctoDash/raw/master/scripts/install.sh)

but it doesn't appear to run anything... just refreshes the screen with a new input line.  Since I know next to nothing about this, does anyone have any ideas on what I might need to try different?  I've looked at various youtube videos, read these instructions, etc. and I'm following them as closely as possible, but nothing.

I _very_ occasionally have an Undervoltage warning... could that be related?

Any help is appreciated - I was really looking forward to using this.

**Comment by FrankX0** [comment]
Well, it’s not you. The installation script (and all other installation files) is not available anymore at that location.
@UnchartedBull can you have a look?

Edit: (intermediate?) fix https://github.com/UnchartedBull/OctoDash/issues/1334#issuecomment-766250313

**Reply by DanDM84** [reply]
Thank you so much!

**Comment by UnchartedBull** [comment, marked as answer]
Yes the fix mentioned in #1334 is the actual fix. I just updatd all the docs to refer to the new main branch. Sorry for the issues.

**Reply by DanDM84** [reply]
Thank you so much!
